src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int supply = 0;
+        int buy = 0;
+
+        try (BufferedReader reader = new BufferedReader(new FileReader(fromFileName))) {
+            String line;
+            while ((line = reader.readLine()) != null) {
+                String[] parts = line.split(",");
+                String operationType = parts[0];
+                int amount = Integer.parseInt(parts[1]);
+                if (operationType.equals("supply")) {
+                    supply += amount;
+                } else if (operationType.equals("buy")) {
+                    buy += amount;
+                }
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
```

Comment:
Avoid opening nested connections to files. Since there are two separate try blocks for reading and writing, consider refactoring the code to separate methods.